Userland/Libraries/LibJS/AST.cpp

```diff
@@ -393,6 +395,101 @@ Value YieldExpression::execute(Interpreter&, GlobalObject&) const
     VERIFY_NOT_REACHED();
 }
 
+// 6.2.3.1 Await, https://tc39.es/ecma262/#await
+static ThrowCompletionOr<Value> await(GlobalObject& global_object, Value value)
```

Comment:
Perhaps this should be moved somewhere else? @linusg might have a stronger opinion on the placement.

Reply:
Moved to `Completion.cpp`